src/spanningtrees/boruvka.jl

```diff
@@ -0,0 +1,56 @@
+@traitfn function boruvka_mst(graph::AG::(!IsDirected), 
+    distmx::AbstractMatrix{T} = weights(graph); 
+    minimize = true) where {T<:Real,U,AG<:AbstractGraph{U}}
+    
+    f_distmx = minimize ? copy(distmx) : copy(-distmx)
+
+    djSet = IntDisjointSets(nv(graph))
+
+    mst = Vector{edgetype(graph)}()
+    sizehint!(mst, nv(graph) - 1)
+
+    weight = 0.0
+    while (num_groups(djSet) > 1)
+
+        cheapest = Array{Union{edgetype(graph), Bool}}(undef, nv(graph))
+
+        for edge in edges(graph)
+            set1 = find_root(djSet, src(edge))
```

Comment:
where is `find_root` defined?

Reply:
> where is `find_root` defined?

It is defined in DataStructures.jl : https://juliacollections.github.io/DataStructures.jl/latest/disjoint_sets/